README.md

```diff
@@ -1,2 +1,117 @@
+Finally, select the Pulumi stack that you want to use. For this repo it is `network-services`.
+```sh
+pulumi stack select network-services
+```
+
+### Making changes
+
+Then preview the changes with:
+```sh
+pulumi preview
+```
+
+### Applying changes
+
+Simply open a PR to see the preview of the changes in the CI. Then, once the PR
+is reviewed and merged into the `main` branch, a new workflow will push the
+changes to Pulumi.
+
+## Changing the DigitalOcean token
+
+Pulumi requires a Personal Access Token (PAT) for DigitalOcean to make calls to the API.
+
+Currently, the PAT is linked to the `ThetaSinner` DigitalOcean user account. To
+change the token, run the following command:
+```sh
+pulumi config set --secret digitalocean:token <new-token>
+```
+
+This value is encrypted by Pulumi and stored in [Pulumi.network-services.yaml].
+
+Remember to open a PR with the new token and allow the CI/Actions to apply the
+changes to Pulumi.
+
+## Bootstrap server
+
+The bootstrap server is a combination of [bootstrap](https://crates.io/crates/kitsune2_bootstrap_srv) and 
+[sbd](https://crates.io/crates/sbd-server) services. Used for peer discovery and initiating peer connections 
+respectively.
+
+The services are deployed as a container which can be found in the [Kitsune2 project](https://github.com/holochain/kitsune2/pkgs/container/kitsune2_bootstrap_srv).
+
+### Setting up on the first deploy
+
+The first step is to set up DNS. You need to map the intended hostname for the server to the IPv4 and IPv6 addresses of
+the server.
+
+Certificates are required to run the bootstrap/sbd service. Generating them with [certbot](https://certbot.eff.org/) is
+an interactive process. You need to run the following command and follow the instructions:
+
+```sh
+sudo certbot certonly --standalone -d bootstrap2.holochain.org
+```
+
+This sets up auto-renewal of certificates, so there's no need to manually configure a cron job. You can check that this 
+is working by running:
+
+```sh
+sudo certbot renew --dry-run
+```
+
+Now you can start the bootstrap/sbd service:
+
+```sh
+cd /opt/bootstrap_srv/
+podman compose up -d
+```
+
+This will pull the `holochain/kitsune2_bootstrap_srv` container and start it as a daemon.
+
+Check that the service managed to start by running:
+
+```sh
+podman compose logs bootstrap
+```
+
+You should see a log line like `#kitsune2_bootstrap_srv#running#`.
+
+### Updating the container deployment
+
+To update the container deployment, edit the `docker-compose.yaml` locally. Any changes to this file should go up for a
+pull request. Once that's done, you need to run the following command locally:
+
+```sh
+scp docker-compose.yaml root@bootstrap2.holochain.org:/opt/bootstrap_srv/docker-compose.yaml
+```
+
+Then, you need to SSH into the server and restart the service:
+
+```sh
+ssh root@bootstrap2.holochain.org
+cd /opt/bootstrap_srv/
+podman compose up -d
+```
+
+Note that this will restart the service, which will close any open connections!
```

Comment:
```suggestion
Note that restarting the service will close any open connections!
```

Reply:
I'm trying to make it clear that updating the image, starts a new container, which means closing connections to the old container. Happy to make this clearer but I think this update doesn't make it clear that the command above does the restart. What do you think?

Reply:
To me it was redundant, because it says "you need to restart the service" followed by the command. It's implicit that it's restarting the service, even though there's no `restart` command.
But I don't mind the redundancy =)